metrics/prometheus/prometheus.go

```diff
@@ -483,6 +497,48 @@ func NewMetrics(cfg config.PrometheusMetrics, disabledMetrics config.DisabledMet
 		"Count of AdsCert request, and if they were successfully sent.",
 		[]string{successLabel})
 
+	metrics.accountDepreciationWarningsPurpose1 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose1_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose1 field")
+	metrics.accountDepreciationWarningsPurpose2 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose2_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose2 field")
+	metrics.accountDepreciationWarningsPurpose3 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose3_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose3 field")
+	metrics.accountDepreciationWarningsPurpose4 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose4_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose4 field")
+	metrics.accountDepreciationWarningsPurpose5 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose5_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose5 field")
+	metrics.accountDepreciationWarningsPurpose6 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose6_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose6 field")
+	metrics.accountDepreciationWarningsPurpose7 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose7_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose7 field")
+	metrics.accountDepreciationWarningsPurpose8 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose8_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose8 field")
+	metrics.accountDepreciationWarningsPurpose9 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose9_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose9 field")
+	metrics.accountDepreciationWarningsPurpose10 = newCounterWithoutLabels(cfg, reg,
+		"account_config_gdpr_tcf2_purpose10_warn",
+		"Count of requests referencing an account whose config specifies a deprecated gdpr.tcf2.purpose10 field")
+
+	metrics.channelEnabledCCPA = newCounterWithoutLabels(cfg, reg,
+		"ccpa_channel_enabled",
+		"Count of requests referencing an account whose config specifies a depreceated ccpa_channel_enabled field")
+	metrics.channelEnabledGDPR = newCounterWithoutLabels(cfg, reg,
+		"gdpr_channel_enabled",
+		"Count of requests referencing an account whose config specifies a depreceated gdpr_channel_enabled field")
+
+	metrics.accountDepreciationSummary = newCounterWithoutLabels(cfg, reg,
+		"account_depreciation_summary",
```

Comment:
Can we change this to `account_config_summary` for now? I could maybe see us recording errors as well as warnings at some point.
I'm finding the name for this one a bit challenging so it would be good to see what another reviewer thinks.